main.js

```diff
@@ -1,129 +1,88 @@
+   * the device.  This the approach used in the Fever Dream / Valence add-on to
+   * find Chrome on Android, for example.
    */
-  connect: function (remotePort) {
-    let port = ConnectionManager.getFreeTCPPort();
-    let local = "tcp:" + port;
-    let remote = "localfilesystem:/data/local/debugger-socket";
-    if (remotePort) {
-      remote = "tcp:" + remotePort;
-    }
-    return adb.forwardPort(local, remote)
-              .then(() => port);
-  },
-
-  type: "adb",
-
-  shell: adb.shell.bind(adb),
-  forwardPort: adb.forwardPort.bind(adb),
-  push: adb.push.bind(adb),
-  pull: adb.pull.bind(adb),
-  reboot: adb.reboot.bind(adb),
-  rebootRecovery: adb.rebootRecovery.bind(adb),
-  rebootBootloader: adb.rebootBootloader.bind(adb),
-
-  isRoot: function() {
-    return adb.shell("id").then(stdout => {
-      if (stdout) {
-        let uid = stdout.match(/uid=(\d+)/)[1];
-        return uid == "0";
-      } else {
-        return false;
-      }
-    });
-  },
-
-  summonRoot: function() {
-    return adb.root();
-  },
 
-  getModel: function() {
-    if (this._modelPromise) {
-      return this._modelPromise;
-    }
-    this._modelPromise = this.shell("getprop ro.product.model")
-                             .then(model => model.trim());
-    return this._modelPromise;
+  function onFastbootStart() {
+    fastboot.startPolling();
   }
-};
-
-/**
- * It may seem at first that registering devices with Device.jsm is no longer
- * needed, since they are now consumed in this same add-on in scanner.js (at
- * least for Firefox 36+ with new WebIDE API).  However, there are still use
- * cases for the Devices.jsm registry, as other add-ons can make use of these
- * low-level ADB devices if they know about other new runtimes to support on the
- * device.  This the approach used in the Fever Dream / Valence add-on to find
- * Chrome on Android, for example.
- */
-
-Devices.on("fastboot-start-polling", fastboot.startPolling.bind(fastboot));
-Devices.on("fastboot-stop-polling", fastboot.stopPolling.bind(fastboot));
+  Devices.on("fastboot-start-polling", onFastbootStart);
+  unload(() => Devices.off("fastboot-start-polling", onFastbootStart));
+  function onFastbootStop() {
+    fastboot.stopPolling();
+  }
+  Devices.on("fastboot-stop-polling", onFastbootStop);
+  unload(() => Devices.on("fastboot-stop-polling", onFastbootStop));
 
-function onConnected(deviceId) {
-  console.log("CONNECTED: " + deviceId);
-  let device = new Device(deviceId);
-  Devices.register(deviceId, device);
-}
-events.on(adb, "device-connected", onConnected);
-unload(() => events.off(adb, "device-connected", onConnected));
+  function onConnected(deviceId) {
+    console.log("CONNECTED: " + deviceId);
+    let device = new Device(deviceId);
+    Devices.register(deviceId, device);
+  }
+  events.on(adb, "device-connected", onConnected);
+  unload(() => events.off(adb, "device-connected", onConnected));
```

Comment:
They look like spaces here... I think GitHub is just highlighting the change in indentation on each line?